fileinfo.py

```diff
@@ -23,17 +22,22 @@
 file_name = raw_input("Enter a file name: ")  # pick a file you have
 count = 0
 t_char = 0
+
 try:
     with open(file_name) as f:
-        line = f.readline()
-        t_char += len(line)
-        while line:
-            count += 1
-            line = f.readline()
-            t_char += len(line)
+        # Source: https://stackoverflow.com/a/1019572
+        count = (sum(1 for line in f))
+        f.seek(0)
+        t_char = (sum([len(line) for line in f]))
 except FileNotFoundError as e:
     print(e)
-    sys.exit()
+    sys.exit(1)
+# When open item is a directory (python2)
```

Comment:
Do we still need to support legacy Python?  https://pythonclock.org/

Reply:
Ha ha, actually because those code was write in  28th November 2011. So may be it should be keep the code still support previous python verison.